src/process_keypress.c

```diff
@@ -72,6 +69,8 @@ void editorProcessKeypress(int fd) {
     case 'l':
       editorMoveCursor(RIGHT);
       break;
+    case 'q':
```

Comment:
added this as a safety for debugging. just incase we break that colon code and want to debug